Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml

```diff
@@ -2804,49 +2939,40 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-  "117":
-    id: "117"
-    taskid: ecd767d9-f7a7-443b-83bb-b6363e3b5c83
+  "129":
+    id: "129"
+    taskid: c92d3355-efa6-4968-8ae0-895cf615fc8a
     type: regular
     task:
-      id: ecd767d9-f7a7-443b-83bb-b6363e3b5c83
+      id: c92d3355-efa6-4968-8ae0-895cf615fc8a
       version: -1
-      name: Send remediation notification email to service owners
-      description: Send an email to service owners about the status of automated remediation action taken.
-      script: '|||send-mail'
+      name: Set playbook stage grid field (decision)
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of command:
```

Comment:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Instead of a value, you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of the command:
```